opentracing-spring-web-itest/common/src/test/java/io/opentracing/contrib/spring/web/interceptor/itest/common/AbstractBaseITests.java

```diff
@@ -87,7 +87,8 @@ public void testAsync() throws Exception {
 
         MockSpan span = mockSpans.get(0);
         Assert.assertEquals("GET", span.operationName());
-        assertLogEvents(span.logEntries(), Arrays.asList("preHandle", "afterCompletion"));
+        assertLogEvents(span.logEntries(), Arrays.asList("preHandle", "afterConcurrentHandlingStarted",
```

Comment:
Note: the additional log entries are because the additional interceptor calls to additional 'preHandle' and 'afterConcurrentHandlingStarted' are now also recorded, as these callback pairs are called within the same thread - i.e. preHandle+afterConcurrentHandlingStarted called in one thread, and preHandle+afterCompletion called in another.